packages/plugins/plugin-sheet/src/testing.tsx

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import type { Decorator } from '@storybook/react';
+import React, { useContext, useEffect, useState } from 'react';
+
+import { Client } from '@dxos/client';
+import { type EchoReactiveObject } from '@dxos/echo-schema';
+
+import { type ComputeGraph, createComputeGraph } from './components';
+import { ComputeGraphContext, ComputeGraphContextProvider } from './components/ComputeGraph/graph-context';
+import { SheetModel } from './model';
+import { createSheet, type CellValue, SheetType } from './types';
+
+export const testSheetName = 'test';
+
+export const createCells = (): Record<string, CellValue> => ({
```

Comment:
i think since /testing is a package entry point it warrants going in a /testing directory and moving the test data into a separate file from the functions, which typically shouldn't be exported.

Reply:
Will add a TODO to find a new home for this.